docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 # TODO(pcattori) Delete this file once RTD fully supports poetry
+recommonmark
```

Comment:
Napoleon ?

Reply:
`napoleon` is already included as a built-in extension to `sphinx`.